### DISCUSSION TITLE
Maintenance on helicopters

### DISCUSSION THREAD

**Discussion by gh658804** [OpenTTD/OpenTTD#10401] [Features]
When setting "Perform maintenance on helicopters at helipads" is On, helicopters are not auto-renewed. I propose that this should either be changed in the background so that helicopters _are_ auto-renewed (sent to depot 6 months after their max yrs) or add an extra setting under the "Perform maintenance" one, something like "but send to depot when it must be auto-renewed"

**Comment by FLHerne** [comment]
This would be fixed by #7111.

I don't agree with @nielsmh's argument in the PR that the "perform maintenance at helipads" setting should be ignored as obsolete.

If it was a `.ini`-file only setting I would, but it can be set through the UI, not just as a compatibility feature for old games.

Unless it's desired to hide this setting from users altogether, it should be expected to work reasonably.
